src/syft/lib/numpy/__init__.py

```diff
@@ -27,10 +27,83 @@ def create_ast(client: TypeAny = None) -> Globals:
     ]
 
     classes: TypeList[TypeTuple[str, str, TypeAny]] = [
-        ("numpy.ndarray", "numpy.ndarray", np.ndarray)
+        ("numpy.ndarray", "numpy.ndarray", np.ndarray),
     ]
 
-    methods: TypeList[TypeTuple[str, str]] = []
+    methods: TypeList[TypeTuple[str, str]] = [
+        # Array attributes
+        ("numpy.ndarray.flags", "numpy.ndarray"),
+        ("numpy.ndarray.shape", "numpy.ndarray"),
+        ("numpy.ndarray.strides", "numpy.ndarray"),
+        ("numpy.ndarray.ndim", "numpy.ndarray"),
+        ("numpy.ndarray.data", "numpy.ndarray"),
+        ("numpy.ndarray.size", "numpy.ndarray"),
+        ("numpy.ndarray.itemsize", "numpy.ndarray"),
+        ("numpy.ndarray.nbytes", "numpy.ndarray"),
+        ("numpy.ndarray.base", "numpy.ndarray"),
+        # ("numpy.ndarray.dtype", "numpy.ndarray"), # SECURITY WARNING: DO NOT ADD TO ALLOW LIST YET
+        ("numpy.ndarray.T", "numpy.ndarray"),
+        # ("numpy.ndarray.real", "numpy.ndarray"), # requires dtype complex
+        # ("numpy.ndarray.imag", "numpy.ndarray"), # requires dtype complex
+        ("numpy.ndarray.flat", "numpy.ndarray"),
+        ("numpy.ndarray.ctypes", "numpy.ndarray"),
+        ("numpy.ndarray.__array_interface__", "numpy.ndarray"),
+        ("numpy.ndarray.__array_struct__", "numpy.ndarray"),
+        # Array methods
+        ("numpy.ndarray.item", "numpy.ndarray"),
+        ("numpy.ndarray.tolist", "numpy.ndarray"),
+        ("numpy.ndarray.itemset", "numpy.ndarray"),
+        ("numpy.ndarray.tostring", "numpy.ndarray"),
+        ("numpy.ndarray.tobytes", "numpy.ndarray"),
+        ("numpy.ndarray.tofile", "numpy.ndarray"),
```

Comment:
I think we might need to remove this if it allows you to write data to any path on the remote system.

Reply:
Sure! On it.